Makefile

```diff
@@ -93,7 +93,8 @@ deb: build
 docs:
 	$(MAKE) -C docs docs
 
-gccgo: build
+gccgo:
+	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)-gccgo" -f "$(DOCKERFILE).gccgo" .
 	$(DOCKER_RUN_DOCKER) hack/make.sh gccgo
```

Comment:
yes, yes it will lol i will update


On Tue, Feb 2, 2016 at 11:10 AM, Tianon Gravi <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/docker/docker/pull/19931#discussion_r51617926>:
>
> > @@ -93,7 +93,8 @@ deb: build
> >  docs:
> >  	$(MAKE) -C docs docs
> >
> > -gccgo: build
> > +gccgo:
> > +	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)-gccgo" -f "$(DOCKERFILE).gccgo" .
> >  	$(DOCKER_RUN_DOCKER) hack/make.sh gccgo
>
> Won't this still be running the wrong image too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19931/files#r51617926>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>